webapp/sources/api-doc/components/schemas/node-add.yml

```diff
@@ -69,7 +69,13 @@ items:
       $ref: agent-key.yml
     properties:
       type: object
-      description: Node properties in "key":"value" format, where "key" is a string, and "value" is either a string, a json array, or a json object.
+      description: >-
+        Node properties in "key":"value" format, where "key" is a string, and "value" is either a string, a json array, or a json object.
+        The following example shows you three example of &#58;
+        &#173; a json array on the example of "tags":[ "some", "tags" ]
+        &#173; a simple string on the example of "env":"prod"
+        &#173; a complex json object on the example of "vars":{ "var1":"value1", "var2":"value2" }
+        They are optional and can be used in any combination as long the keys are unique within the properties object.
```

Comment:
The description doesn't accept break line, the is one whole paragraph, I doesn't find a solution to make it more readable from the UI